### ISSUE TITLE
A11y_GradeB_VSCode_Unable to find Rating - Blind:  Tab navigation to Rating link in the extension its name as "rating" and the visual rating is not read

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#23251]
**GitHubTags**:-A11ySev2;-A11yTCS;-A11yMAS;-DesktopApp;-Visual Studio Code Client;-WCAG1.3.1;-DesktopApp;-BM-VisualStudioCode-Win32-Nov2021;-NVDA;-JAWS;-Win32;-Win11;-AINotInScope;

-- Environment Details:​
Application Name: Visual Studio Code
Version: 1.79.0-insider (user setup)
OS build	25300.1000
JAWS 2023.2212.23 ILM

-- Repro Steps:

1. Launch Visual Studio Code insiders
2. Navigate to extension and search for "Javascript"
3. start JAWS
4. tab to the rating section at the top 

-- Actual:
Tab navigation to Rating link in the extension its name as "rating" and the visual rating is not read, It reads left right parenthesis the number and control as link
and in speech viewer we can see the text read 

-- Expected:
Tab navigation to Rating link in the extension it should be read as average ration 4 of out 5 link 540

--User Impact:
screen reader users will not know the rating of the extension 

-- Attachment
[Unable to find rating-JAWS.webm](https://github-com/microsoft/vscode/assets/24580570/14d05f11-cd09-4840-a0bc-224ce6d09947)
